diagnostic.md

```diff
@@ -32,23 +35,27 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+.val()
+.val(value)
+.html
+.html(html)
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+A representation of document that allows it to be accessed from other programs in order for those programs to change it.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.